coast/NEMO.py

```diff
@@ -268,7 +273,83 @@ def transect_indices(self, start: tuple, end: tuple) -> tuple:
         ii1 = [int(ii) for ii in np.round(np.linspace(i1, i2, num=line_length))]
         
         return jj1, ii1, line_length
+    
+    
+    def construct_density( self, EOS='EOS10' ):
+        
+        '''
+            Constructs the density using the salinity, temperture and depth_0 fields
+            and adds a density attribute to the dataset 
+            
```

Comment:
Say something about "absolute salinity", "conservative temperature", and "absolute pressure" in the doc string? I.e. make these terms greppable.